modules/javafx.graphics/src/main/java/com/sun/glass/ui/gtk/GtkWindow.java

```diff
@@ -187,22 +187,8 @@ public long getNativeWindow() {
     protected void _setBounds(long ptr, int x, int y, boolean xSet, boolean ySet, int w, int h, int cw, int ch, float xGravity, float yGravity) {
         _setGravity(ptr, xGravity, yGravity);
         setBoundsImpl(ptr, x, y, xSet, ySet, w, h, cw, ch);
-
-        if ((w <= 0) && (cw > 0) || (h <= 0) && (ch > 0)) {
```

Comment:
Why is this block removed?

Reply:
Size adjusting accounting frame extents is done in `WindowContextTop::update_frame_extents()`

Reply:
I mainly wanted to make sure we were not missing a `notifyResize` event (which was executed in the removed block), but I see this is now done via `WindowContextTop::set_bounds` which is fine.